### PR TITLE
linting / formatting cleanup

### PR DESCRIPTION
**Pull request by willscott**
* all logging is pair'ed as key/value pairs
* usage of slog is removed
* comments addded to make linting happy

### CODE REVIEW

**Comment by willscott** [comment]
this is failing in some form of JSON decoding:
```
[+] Nodes finished running DKG. Checking keys...
[+] Checking if chain info is present on all nodes...
invalid character 's' in literal true (expecting 'r')
[+] Printing logs for debugging on CI
```
The error line is run by the `subprocess` node implementation of the demo integration test. It appears to be attempting to parse a `true`, and has gotten past the `t`, and is now expecting an `r` but instead sees an `s`.... unclear what the root cause is here, since nothing in this PR seems like it'll trigger that.

**Comment by willscott** [comment]
yep :)

**Comment by willscott** [comment]
well - the gossip relay still uses the ipfs logger

**Comment by willscott** [comment]
it does get us within arms reach of being able to run a linter in CI as well

**Comment by willscott** [comment]
error is due to a log that's now output by the sub-drand command
```
err json decoding:
ts="Tue, 26 May 2020 06:41:48 UTC" call=cli.go:317 level=info cert_manager=add servercertpath=/tmp/drand-full/node-1/server-1.crt
{
    "public_key": "b9392f2647078b8d656ab2731aaef906e5b1cd65b234fe74dc91eaeb8cf41e93f3859215908abb09dc7ba8476effa8c1",
    "period": 10,
    "genesis_time": 1590475330
}
```

**Comment by willscott** [comment]
current issue is:
```
curl output: {"round":2,"signature":"af70ab6e5effa25db51ff845323a7d8b99bd1d6ab25ada3950cdd20674402f00dcb9f6397011d4ce84291cb38e0e8eff05328c5de4d7e366e15c285bbdb087639fa79fa912a9c61eb265b6acdaf74adbada3bcc3e52f5b966ef89d5c02e720a4","previous_signature":"81a15c59d91ca6dda68be308a1119c9e79caa2f479bf501bd91ffb1c58ccb9f70dcdb1b1a55abf6f7e32ca246796f561118ec592ad63659d0c2438cf6c1f44b9d329083202560f5bcec264f9b9e13732560db2a11a2b454c2fba73430ca2a749","randomness":"872bd28283d26ca9931de3a8e8a639b8e5b903fa13c9cb733fb3ae64101d4396"}

cli output: round:2 signature:"\257p\253n^\377\242]\265\037\370E2:}\213\231\275\035j\262Z\3329P\315\322\006t@/\000\334\271\3669p\021\324\316\204)\034\263\216\016\216\377\0052\214]\344\327\343f\341\\([\275\260\207c\237\247\237\251\022\251\306\036\262e\266\254\332\367J\333\255\243\274\303\345/[\226n\370\235\\\002\347 \244" previous_signature:"\201\241\\Y\331\034\246\335\246\213\343\010\241\021\234\236y\312\242\364y\277P\033\331\037\373\034X\314\271\367\r\315\261\261\245Z\277o~2\312$g\226\365a\021\216\305\222\255ce\235\014$8\317l\037D\271\323)\0102\002V\017[\316\302d\371\271\34172V\r\262\241\032+EL/\272sC\014\242\247I" randomness:"\207+\322\202\203\322l\251\223\035\343\250\350\2469\270\345\271\003\372\023\311\313s?\263\256d\020\035C\226" 

[-] Inconsistent signature from curl vs CLI
```

File: chain/chain.go [hunk 1]
```diff
@@ -172,7 +172,7 @@ func (c *chainStore) runChainLoop() {
 			c.l.Fatal("new_beacon_storing", err)
 		}
 		lastBeacon = newB
-		c.l.Info("NEW_BEACON_STORED", newB.String())
+		c.l.Info("new_beacon_stored", newB.String())
```
Comment (nikkolasg) [inline chain/chain.go:175]:
```sugesstion
NEW_BEACON_STORED
```

**Pull request merged**